backend/content/models.py

```diff
@@ -25,27 +27,28 @@
 class Faq(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     org_id = models.ForeignKey("entities.Organization", on_delete=models.CASCADE)
-    name = models.CharField(max_length=255)
     question = models.TextField(max_length=500)
     answer = models.TextField(max_length=500)
     last_updated = models.DateTimeField(auto_now=True)
 
     def __str__(self) -> str:
-        return self.name
+        return self.question
 
 
 class Resource(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     name = models.CharField(max_length=255)
     description = models.TextField(max_length=500)
-    topics = ArrayField(models.CharField(max_length=255), default=list, blank=True)
-    category = models.CharField(max_length=255, blank=True)
+    topics = ArrayField(
+        models.CharField(max_length=255), default=list, blank=True, null=True
```

Comment:
CharField and TextField are never saved as NULL in django therefore null=True is not necessary.

In case you want to read a bit more about here is a good source https://stackoverflow.com/questions/8609192/what-is-the-difference-between-null-true-and-blank-true-in-django